### PR TITLE
Rename @aspnet/signalr to @microsoft/signalr

### LINKED ISSUE #11568: Rename '@aspnet/signalr' and '@aspnet/signalr-protocol-msgpack' to '@microsoft/...'
We have [the name reserved on NPM](https://www.npmjs.com/package/@microsoft/signalr) and want to do this rename to help clarify that the SignalR client can be used in scenarios with the Azure SignalR Service where there is no ASP.NET :). We need to rename our `package.json` files and update dependencies to do this.

@pranavkm @rynowak This shouldn't affect Blazor as you embed the client, right? We'll have to update the build logic to make sure it works (we'll do the needful) but there shouldn't be a runtime impact from my understanding

fyi @DamianEdwards @bradygaster @davidfowl

### PR DESCRIPTION
**Pull request by mikaelm12**
Fixes: https://github.com/aspnet/AspNetCore/issues/11568
Replacing all instances of `@aspnet` with `@microsoft`. Are there any other tricks here?
Follow up items are to update docs and publish preview 7 under the `@aspnet/signalr` name with a deprecated message

### CODE REVIEW

**Comment by anurse** [comment]
Don't see any blazor-related changes, I presume they weren't using the ID anywhere, just the files?

Also, speaking of docs, we should sync up on the JS API docs. They'll probably need to be updated.

**Review by analogrelay — APPROVED** [review]
Looks right to me, as long as the build passes and things are working, I'm good with this.

File: src/SignalR/clients/ts/FunctionalTests/web.config [hunk 1]
```diff
@@ -7,8 +7,11 @@
     <handlers>
       <add name="aspNetCore" path="*" verb="*" modules="AspNetCoreModuleV2" resourceType="Unspecified" />
     </handlers>
-    <aspNetCore processPath="%LAUNCHER_PATH%" arguments="%LAUNCHER_ARGS%" stdoutLogEnabled="false" stdoutLogFile=".\logs\stdout" forwardWindowsAuthToken="false" startupTimeLimit="3600" requestTimeout="23:00:00">
```
Comment (mikaelm12) [inline src/SignalR/clients/ts/FunctionalTests/web.config]: I'll revert this

**Comment by mikaelm12** [comment]
> Looks right to me, as long as the build passes and things are working, 

Just need to do the needful for the Code check and everything should be green

File: src/SignalR/clients/ts/FunctionalTests/web.config (hunk 1 above)
Comment (BrennanConroy) [reply src/SignalR/clients/ts/FunctionalTests/web.config]: Forget?

Comment (mikaelm12) [reply src/SignalR/clients/ts/FunctionalTests/web.config]: I was making the change as you were typing this lol

**Review by BrennanConroy — APPROVED** [review]

**Comment by benaadams** [comment]
@anurse  This needs an announcement?

**Comment by anurse** [comment]
Yep, it will have one!

**Comment by anurse** [comment]
https://github.com/aspnet/Announcements/issues/372

**Comment by mikaelm12** [comment]
Thanks @anurse!

**Pull request merged**